src/integrationTest/groovy/com/streamr/client/DataUnionClientSpec.groovy

```diff
@@ -0,0 +1,97 @@
+package com.streamr.client
+
+import com.streamr.client.dataunion.DataUnion
+import com.streamr.client.dataunion.DataUnionClient
+import com.streamr.client.dataunion.EstimatedGasProvider
+import com.streamr.client.dataunion.EthereumTransactionReceipt
+import com.streamr.client.dataunion.contracts.IERC20
+import org.web3j.abi.datatypes.*
+import org.web3j.abi.datatypes.generated.Uint256
+import org.web3j.crypto.Credentials
+import org.web3j.protocol.Web3j
+import org.web3j.protocol.core.methods.response.TransactionReceipt
+import org.web3j.protocol.http.HttpService
+
+class DataUnionClientSpec extends StreamrIntegrationSpecification{
+    private DataUnionClient client
+    //truffle keys mnemonic "testrpc"
+    private String[] testrpc_keys = [
+        "0x5e98cce00cff5dea6b454889f359a4ec06b9fa6b88e9d69b86de8e1c81887da0",
+        "0xe5af7834455b7239881b85be89d905d6881dcb4751063897f12be1b0dd546bdb",
+        "0x4059de411f15511a85ce332e7a428f36492ab4e87c7830099dadbf130f1896ae",
+        "0x633a182fb8975f22aaad41e9008cb49a432e9fdfef37f151e9e7c54e96258ef9",
+        "0x957a8212980a9a39bf7c03dcbeea3c722d66f2b359c669feceb0e3ba8209a297",
+        "0xfe1d528b7e204a5bdfb7668a1ed3adfee45b4b96960a175c9ef0ad16dd58d728",
+        "0xd7609ae3a29375768fac8bc0f8c2f6ac81c5f2ffca2b981e6cf15460f01efe14",
+        "0xb1abdb742d3924a45b0a54f780f0f21b9d9283b231a0a0b35ce5e455fa5375e7",
+        "0x2cd9855d17e01ce041953829398af7e48b24ece04ff9d0e183414de54dc52285",
+        "0x2c326a4c139eced39709b235fffa1fde7c252f3f7b505103f7b251586c35d543"
+    ]
```

Comment:
There are actually only 3 test users in this tests suite: 1) DU admin-agent, 2) joining member, 3) withdraw recipient. Could create three variables with descriptive names and remove the obsolete private keys?

Reply:
I would leave. These are the same 10 keys we list in every test.